### DISCUSSION TITLE
How to properly use this theme?

### DISCUSSION THREAD

**Discussion by twatzl** [nusserstudios/tailbliss#98] [Q&A] [answered]
Ok. So I could not really find a good documentation on how to use this theme. How is it intended to be used?

I don't find a reference of config parameters and from what I looked at the code it seems everything is hardcoded. E.g. for the social media the icons are hardcoded paths? 
So it's not possible to extend with e.g. LinkedIn or Pixelfed unless I also add the logos as path. 

This seems weird to me as it defeats the whole purpose of having a hugo theme. Also it doesn't really match to the advertise "flexibility" of the theme. 

Do I miss some config options here or am I really supposed to edit the partials?

**Comment by nusserstudios** [comment, marked as answer]
Yeah, it's a Hugo site more than a theme. Heavy Frontend. Probably not going to use Hugo anymore as I've switched to Astro, Next and Qwik.

**Reply by SnapshotCiTy** [reply]
does it mean it's abandoned?

**Reply by nusserstudios** [reply]
Im going to keep updating it, but if someone does. a pull request to make it better, I'd gladly accept. So no, it isn't abandoned, but I also wont be putting much time and energy into it, either.
